src/ui/widgets/CategoriesStack/styles.ts

```diff
@@ -0,0 +1,21 @@
+import { StyleSheet } from 'react-native';
+
+export const stylesheet = StyleSheet.create({
+  inlineRow: {
+    flexDirection: 'row',
+    width: '100%'
+  },
+  smallCard: {
+    width: '37%',
```

Comment:
We will not specify width

Reply:
Grid should be set such , Card will take parents grid size

Reply:
Somewhere we need to specify the width. Else it would take the whole width and divide the whole by number of contents inside it. I tried with Grid from this library. It doesn't allow children. 
Either we will sacrifice the design and make it equal length for all which is easily doable. Else we have to setup width. If you are thinking other way, please let me know.